test/sql/storage/attach_mixed_numeric.test

```diff
@@ -0,0 +1,33 @@
+# name: test/sql/storage/attach_mixed_numeric.test
+# description:
+# group: [sqlite_storage]
+
+require sqlite_scanner
+
+# load from a database with a numeric column that has mixed types
+
+# using the all_varchar option, without the setting
```

Comment:
Your implementation is different to mine, so you should update these comments

Reply:
Yes good point, done